app/src/pages/playground/PlaygroundDatasetSection.tsx

```diff
@@ -195,23 +180,9 @@ export function PlaygroundDatasetSection({
                   ])}
               </Flex>
               <PlaygroundEvaluatorSelect
-                evaluators={
-                  evaluators as Mutable<(typeof evaluators)[number]>[]
-                }
+                evaluators={evaluators}
                 selectedIds={selectedEvaluatorIds}
-                onSelectionChange={(id: string) => {
-                  const evaluator = evaluators.find((e) => e.id === id);
-                  if (evaluator?.isAssignedToDataset) {
-                    setSelectedEvaluatorIds((prev) => {
-                      if (prev.includes(id)) {
-                        return prev.filter((evaluatorId) => evaluatorId !== id);
-                      }
-                      return [...prev, id];
-                    });
-                  } else {
-                    setAddingEvaluatorId(id);
-                  }
-                }}
+                onSelectionChange={setSelectedEvaluatorIds}
                 addNewEvaluatorLink={prependBasename(`/evaluators/new`)}
```

Comment:
this page just got removed, we can probably just not show the link for now or you can link them to the dataset/:id/evaluators page